source/doc-odm-user-guide/setup.rst

```diff
@@ -60,9 +60,8 @@ of that group. Any user can create one or several groups and invite their collab
 
 Depending on the role a users has, they can have different privileges in sharing process and managing the group, namely:
 
-- *Non-sharing* — can only view the files shared with the group, however they have no sharing permissions;
-- *Sharing User* — not only have an access to the shared files but also can share data themselves;
-- *Group Administrator* — in addition to the sharing user rights, can invite or remove users and change their privileges.
+- *Group member* — have an access to the shared files but also can share data themselves;
```

Comment:
I suggest adding info that anyway only owner can share study and if user belongs to any group - he can share study with this group